content/r/concepts/comments/comments.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: 'Comments'
+Description: 'A comment is a piece of text within a program that is not executed. It can be used to provide additional information to aid in understanding the code.'
```

Comment:
I think this definition can be expanded a bit. Comments (commenting) can serve multiple purposes in addition to clarifying code, such as commenting out a function or section of code during debugging